packages/main/src/themes/SegmentedButton.css

```diff
@@ -36,14 +36,14 @@
 	background-color: var(--sapButton_Hover_Background);
 }
 
-::slotted([ui5-segmented-button-item][pressed]),
+::slotted([ui5-segmented-button-item][selected]),
```

Comment:
Now as we have a `SegmentedButtonItem.css` file, wouldn't it be better to transfer those styles here ?

Reply:
some of them